scipy/constants/tests/test_codata.py

```diff
@@ -51,7 +51,27 @@ def test_2002_vs_2006():
 
 def test_exact_values():
     # Check that updating stored values with exact ones worked.
-    with suppress_warnings() as sup:
-        sup.filter(ConstantWarning)
-        for key in _cd.exact_values:
-            assert_((_cd.exact_values[key][0] - value(key)) / value(key) == 0)
+    exact = dict((k, v[0]) for k, v in _cd._physical_constants_2018.items())
+    replace = _cd.exact2018(exact)
+    for key, val in replace.items():
+        assert_equal(val, value(key))
```

Comment:
I think that in this case `value(key)` is the actual value and `val` is the reference, but it's `assert_equal`, so practically speaking, the order doesn't matter.